README.md

```diff
@@ -142,6 +142,7 @@ require("babylon").parse("code", {
  - `numericSeparator` ([proposal](https://github.com/samuelgoto/proposal-numeric-separator))
  - `optionalChaining` ([proposal](https://github.com/tc39/proposal-optional-chaining))
  - `importMeta` ([proposal](https://github.com/tc39/proposal-import-meta))
+ - `bigint` ([proposal](https://github.com/tc39/proposal-bigint))
```

Comment:
I actually started with the plugin named bigInt. But then when it came time to make a new token type, I saw that the `RegExp` constructor mapped to `regex` so I figured that `BigInt` should map to `bigint`. So then I changed the plugin to match the token type.

But I'm ok with `bigInt` if you think that's better